addons/sys_zeus/fnc_handleZeusSpeakPress.sqf

```diff
@@ -0,0 +1,15 @@
+/*
+ * Author: SynixeBrett
+ * Handles the speak from zeus camera press down
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+if (call EFUNC(sys_core,inZeus) && {!ACRE_IS_SPECTATOR}) then {
+    GVAR(speakFromZeusHandle) = [{
+        player setVariable [QGVAR(inZeus), true, true];
```

Comment:
Do those variables really have to be synced over network?

Reply:
Ah I see. That's quite a lot of network traffic. Could we combine those 3 datas into one for starters?

Reply:
I noticed `inZeus` can be moved outside perFrameHandler, and I've combined the other two in one